modules/administration-guide/partials/proc_backing-up-a-che-instance-by-using-the-checlusterbackup-custom-object.adoc

```diff
@@ -0,0 +1,67 @@
+[id="backing-up-a-{prod-id-short}-instance-by-using-the-checlusterbackup-custom-object_{context}"]
+= Backing up a {prod-short} instance by using the CheClusterBackup custom object
+
+You can use a `CheClusterBackup` custom object to make a backup snapshot of a {prod-short} instance and to send the snapshot to the configured backup server. To create each and every backup snapshot requires a new `CheClusterBackup` custom object; that is, editing an already consumed `CheClusterBackup` custom object, during or after backing up, has no effect.
+
+include::snip_warning-does-not-back-up-content-of-workspaces.adoc[]
+
+.Prerequisites
+
+* Configured backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+* Created `CheBackupServerConfiguration` custom object. See the previous section of this guide.
+
+.Procedure
+
+. Create the `CheClusterBackup` custom object, which creates a backup snapshot:
++
+[source,yaml,subs="+quotes,+attributes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheClusterBackup
+metadata:
+  name: {prod-id}-backup
+  namespace: {prod-namespace}
+spec:
+  backupServerConfigRef: backup-server-configuration <1>
+  useInternalBackupServer: false <2>
+----
+<1> Name of the `CheBackupServerConfiguration` custom object defining what backup server to use.
+<2> Configures the custom resource to back up to a backup server.
```

Comment:
I'd rephrase because it backup to a backup server anyway. The question is between internal backup server managed by Che or a server (internal or external) managed by admin.
nitpick: it configures operator via this CR, not CR itself

Reply:
@mmorhun, I've changed it to:
```
<2> Configures the Operator via this custom resource 
to use the {prod-short}-managed internal backup server 
or an administrator-managed external backup server 
(SFTP, Amazon S3 or S3 API compatible storage, or REST).
```